README.md

```diff
@@ -34,7 +34,7 @@ git branch --show-current
 
 This command shows the current branch you are currently on. This is useful if you are ever unsure. You should currently be on 'main', which is the default branch often used. However, once verified, we want to switch to our own. Please do so with:
 
-git branch -checkout 'jay-leach'
+git -checkout 'jay-leach'
```

Comment:
This isn't correct, please reference creating a new branch and NOT JUST switching from an existing one.